docs/learn/fundamentals/transactions/unique-source-for-operation.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "When Operation Source Accounts differ from that of the Parent Transaction"
+sidebar_label: Unique Source Accounts for Operations
+description: "Learn"
```

Comment:
This seems a little vague for a description? were you meaning for this to be "Learn" or was it a placeholder maybe?